Tests/Fake/TestData.swift

```diff
@@ -65,4 +65,19 @@ public enum TestData {
             return nil
         }
     }
+
+    private static let ghRestOperations: [String: Any] = {
+        let data = resource(named: "ghRestOperations.json")
+        let object = try! JSONSerialization.jsonObject(with: data, options: [])
+        return object as! [String: Any]
+    }()
+
+    /// Probably could be more efficient than encoding then decoding again?!
```

Comment:
Not unless you encode each individual operation as a data blob in the original JSON, which defeats the purpose of making it so conveniently structured. Not a big deal. I might recommend you do the re-serialize step as part of the "loading" step above, though, so that you end up with `[String: Data]`; that way you shift the all of the overhead into the setup stage rather than having it affect each test's execution time (normally you'd _want_ that kind of amortization, of course, but for tests an accurate performance measurement (even in a debug build) is generally more useful).